PRODUCTION_UPGRADE.md

```
@@ -0,0 +1,349 @@
+```bash
+# Set environment
+export PROXY_ADDRESS=0x97b2cb568e0880B99Cd16EFc6edFF5272Aa02676
+export PROXY_ADMIN_ADDRESS=0xECE9bE2e4b0c9a2C9E305feA6Ead25d310477409
+export MAINNET_RPC=your_mainnet_rpc_url
+
+# Test on fork (SAFE - no real transactions)
+yarn upgrade:mainnet:simulate
+```
+
+**Expected Output:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)